modules/fundamental/src/advisory/model/details/mod.rs

```diff
@@ -40,23 +42,29 @@ impl AdvisoryDetails {
                 entity::advisory_vulnerability::Column::VulnerabilityId,
                 entity::vulnerability::Column::Id,
             )
-            .filter(entity::advisory_vulnerability::Column::AdvisoryId.eq(advisory.id))
+            .filter(entity::advisory_vulnerability::Column::AdvisoryId.eq(advisory.advisory.id))
             .all(tx)
             .await?;
 
         let vulnerabilities =
-            AdvisoryVulnerabilitySummary::from_entities(advisory, &vulnerabilities, tx).await?;
-
-        let issuer = advisory
-            .find_related(entity::organization::Entity)
-            .one(tx)
-            .await?;
+            AdvisoryVulnerabilitySummary::from_entities(&advisory.advisory, &vulnerabilities, tx)
+                .await?;
 
         Ok(AdvisoryDetails {
-            head: AdvisoryHead::from_advisory(advisory, Memo::Provided(issuer), tx).await?,
+            head: AdvisoryHead::from_advisory(
+                &advisory.advisory,
+                Memo::Provided(advisory.issuer.clone()),
+                tx,
+            )
+            .await?,
+            source_document: if let Some(doc) = &advisory.source_document {
+                Some(SourceDocument::from_entity(doc, tx).await?)
+            } else {
+                None
+            },
```

Comment:
This is a breaking api change, right? I can't decide whether the `source_document` element is important enough to clients to have it broken out.

Reply:
Adjusted to flatten into the owning DTO.